### DISCUSSION TITLE
[FR] Disable updating the current entry directly, force using a draft in multi site environments

### DISCUSSION THREAD

**Discussion by wsydney76** [craftcms/cms#8928] [Ideas]
### Description

This is the most important point from #6687 

Having a multi site setup with non translatable fields, or matrix fields set to 'save blocks to all sites', saving the current entry may lead to inconsistent content on the other sites.

Example

Non translatable image: __a dog__ - Text en: __This is a dog__ - Text fr: __C'est un chien__

Update image and text in french in current entry and save:

Non translatable image: __a cat__ - Text en: __This is a dog__ - Text fr: __Ceci est un chat__

Which is obviously a false biological information on the english site (at least for those still caring about facts). Sooner or later it will happen to even the most attentive editor. And it will be discoverd, even if it is repaired quickly.

So it would be nice to have a setting for a section to lock the current version (similar to revisions),

As the HUD editors also update the current version, this should also not be possible in this case.

**Comment by brandonkelly** [comment]
Related: #6022

**Comment by wsydney76** [comment]
Cleaning up, this is no longer an issue.
